gists/pull-request-guidelines.md

```diff
@@ -0,0 +1,69 @@
+# Pull Request (PR) Guidelines
+
+Guidelines for creating pull requests at LaunchPad Lab for three types of users: authors, code reviewers, and QA reviewers.
+
+- **Author**: The creator of the pull request.
+- **Code Reviewer**: A developer assigned to the pull request to review the new and/or modified code, not the application's functionality.
+- **QA Reviewer**: An individual assigned to the pull request to not only test the new and/or updated functionality introduced by the pull request, but also to raise any issues found related to existing functionality.
+
+## Authors
+- If there is an existing [pull request template](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository), follow the steps and comments included
+  - These templates will come out-of-the-box when starting a new project using LPL's internal starting applications
+- Convert a PR to a [draft pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) if it is not ready for review yet (useful for [long-running](#long-running) prs)
```

Comment:
Nice! I was curious how we wanted to adopt this. Perhaps if PRs are done in another tool (ex. Bitbucket) the fallback is to include `[DRAFT]` in the PR title.